.github/workflows/autoupdate.yml

```
@@ -0,0 +1,60 @@
+    permissions:
+      contents: write
+      pull-requests: write
+      
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.PAT_FOR_CROSS_REPOS_CICD_TRIGGERING }}  # Personal Access Token with repo scope
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)